proto/regen/ecocredit/v1alpha1/events.proto

```diff
@@ -53,17 +53,22 @@ message EventReceive {
   // initial issuance or taken from a basket.
   string sender = 1;
 
+  // basket_denom is the denom of the basket. when the basket_denom field is set, it indicates that this event was
+  // triggered by the transfer of credits from a basket. It will not be set if the credits were sent by a user, or
+  // by initial issuance.
+  string basket_denom = 2;
+
   // recipient is the recipient of the credits.
-  string recipient = 2;
+  string recipient = 3;
```

Comment:
let's not do any proto breakage here please. this is already in production. `basket_denom` should be `6`

Reply:
fixed